Makefile

```diff
@@ -323,6 +323,10 @@ import-manifests: kustomize
 		$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone $(IMPORT_MANIFESTS_PATH)/config/overlays/mutation -o $(GATEKEEPER_MANIFEST_DIR); \
 	fi
 
+	sed -i '73,80d' $(GATEKEEPER_MANIFEST_DIR)/rbac.authorization.k8s.io_v1_clusterrole_gatekeeper-manager-role.yaml
```

Comment:
Specifying line numbers is prone to error if there's an update. Could we use a `yq` command here instead? Maybe something like this?
```
yq e '.rules[] |= del(select(.resources[] == "podsecuritypolicies"))' -i $(GATEKEEPER_MANIFEST_DIR)/rbac.authorization.k8s.io_v1_clusterrole_gatekeeper-manager-role.yaml
```
Please also add a brief comment here on why it's necessary.

Reply:
Neat!